rosys/vision/rtsp_camera/jovision_rtsp_interface.py

```diff
@@ -112,11 +114,12 @@ def get_current_settings(self) -> list[JovisionCameraSettings]:
                 'channelid': 0,
             },
         }
-        params = {
-            'cmd': json.dumps(cmd),
-            '_': int(time.time() * 1000),
-        }
-        response = requests.get(self.settings_url, params=params, timeout=1)  # type: ignore
+        params = QueryParams(
+            cmd=json.dumps(cmd),
+            _=int(time.time() * 1000),
+        )
+        async with httpx.AsyncClient() as client:
```

Comment:
Generally it is better to instantiate the client in the init function to reduce overhead.